extensions/rvdatabinding/src/main/java/com/google/android/agera/rvdatabinding/RecycleConfig.java

```diff
@@ -51,8 +52,13 @@
   int CLEAR_HANDLERS = 1 << 1;
 
   /**
-   * When the {@link RecyclerView} recycles a view, rebind both the item from the {@link Repository}
-   * and all handlers with {@code null}.
+   * When the {@link RecyclerView} recycles a view, rebind the collection from the
```

Comment:
reset the collection variable to null.

Similar changes to other wording: "reset the item variable to null"; "reset the handler variables to null"; "reset all variables to null".